### PR TITLE
Add common independent physicians

### PR DESCRIPTION
**Pull request by westnordost**
Adds common independent physicians as selectable presets.

### CODE REVIEW

File: data/presets/amenity/doctors/general.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "addTags": {
+        "healthcare": "doctor"
+    },
+    "reference": {
+        "key": "healthcare:speciality",
+        "value": "general"
+    },
+    "name": "Primary care physician",
+    "aliases": [
+        "General practitioner"
```
Comment (arch0345) [inline data/presets/amenity/doctors/general.json:26]:
```suggestion
    "name": "Primary Care Physician",
    "aliases": [
        "General Practitioner"
```

File: data/presets/amenity/doctors/general.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "icon": "maki-doctor",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "GP",
+        "Family Physician",
+        "Family Practitioner",
+        "Family Doctor"
```
Comment (arch0345) [inline data/presets/amenity/doctors/general.json:11]:
```suggestion
        "gp",
        "family physician",
        "family practitioner",
        "family doctor"
```

File: data/presets/amenity/doctors/orthopaedics.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+    "icon": "maki-doctor",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "Orthopedic surgery"
```
Comment (arch0345) [inline data/presets/amenity/doctors/orthopaedics.json:8]:
```suggestion
        "orthopedic surgery"
```

File: data/presets/amenity/doctors/otolaryngology.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    "terms": [
+        "ear",
+        "ENT",
+        "head and neck surgery",
+        "nose",
+        "OHNS",
+        "ORL",
+        "ORL–H&N",
+        "Otolaryngologist",
+        "Otorhinolaryngology",
```
Comment (arch0345) [inline data/presets/amenity/doctors/otolaryngology.json:16]:
```suggestion
        "ent",
        "head and neck surgery",
        "nose",
        "ohns",
        "orl",
        "orl–h&n",
        "otolaryngologist",
        "otorhinolaryngology",
```

File: data/presets/amenity/doctors/paediatrics.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    "icon": "maki-doctor",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "adolescents",
+        "children",
+        "infants",
+        "Paediatrics"
```
Comment (arch0345) [inline data/presets/amenity/doctors/paediatrics.json:11]:
```suggestion
        "paediatrics"
```

File: data/presets/amenity/doctors/pathology.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
+{
+    "icon": "maki-doctor",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "autopsy",
+        "Anatomical pathology",
+        "Anatomic pathology",
```
Comment (arch0345) [inline data/presets/amenity/doctors/pathology.json:10]:
```suggestion
        "anatomical pathology",
        "anatomic pathology",
```

File: data/presets/amenity/doctors/proctology.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
+{
+    "icon": "maki-doctor",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "assman",
+        "Colorectal surgery",
+        "Proctology"
```
Comment (arch0345) [inline data/presets/amenity/doctors/proctology.json:10]:
```suggestion
        "colorectal surgery",
        "proctology"
```

File: data/presets/amenity/doctors/surgery.json [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+        "healthcare:speciality": "surgery"
+    },
+    "addTags": {
+        "healthcare": "doctor"
+    },
+    "reference": {
+        "key": "healthcare:speciality",
+        "value": "surgery"
+    },
+    "name": "General surgeon"
```
Comment (arch0345) [inline data/presets/amenity/doctors/surgery.json:21]:
```suggestion
    "name": "General Surgeon"
```

File: data/presets/amenity/doctors/trauma.json [hunk 9]
```diff
@@ -0,0 +1,23 @@
+{
+    "icon": "maki-doctor",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "Surgery",
+        "Trauma surgery"
```
Comment (arch0345) [inline data/presets/amenity/doctors/trauma.json:9]:
```suggestion
        "surgery",
        "trauma surgery"
```

File: data/presets/amenity/doctors/rheumatology.json [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+    "icon": "maki-doctor",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "lupus",
+        "oteoporosis",
+        "rheuma",
+        "Rheumatology"
```
Comment (arch0345) [inline data/presets/amenity/doctors/rheumatology.json:11]:
```suggestion
        "rheumatology"
```

**Review by arch0345 — CHANGES_REQUESTED** [review]
`terms` are typically all lowercase while `names` and `aliases` are typically in title case

**Comment by tyrasd** [comment]
Merged as d40b047, as there was a small mistake in these presets: `addTags` needs to repeat the main tags of a preset in most cases. See https://github.com/ideditor/schema-builder#addtags

**Comment by westnordost** [comment]
Oh, I see. Thank you.

**Pull request closed without merging**